extending-dummy4j.md

```diff
@@ -39,21 +39,32 @@ files of the same name within the same path may override each other.
 
 If you choose to, you can place all of your definitions in one file. You can also spread them out over several files.
 
-## Parser
+## Expression resolver
 
-The parser recognizes two placeholders:
-* `#{key.path}` - will resolve to a random value from the list of data definitions using the provided key path; The
- resolved value may itself be an expression
-* `#` - will resolve to a random digit between 0 and 9 
+The expression resolver recognizes three placeholders:
+* `#{path.to.key}` (single-locale placeholder) - will resolve to a random value from the list of data definitions using
+ the provided key path; The resolved value may itself be an expression. If the path resolves to a list of keys
```

Comment:
- capital letter after semicolon:
`key path; The resolved` to `key path; the resolved`